src/Classes/Job/Blender/Execute.php

```diff
@@ -49,6 +55,61 @@ public function create(array $jobObject): bool
         return parent::create($jobObject);
     }
 
+    /**
+     * @param array $config
+     *
+     * @return bool
+     *
+     * @throws Exception
+     */
+    public function run(array $config): bool
+    {
+        parent::run($config);
+
+        $replicas = 1;
+        $count = App::getDbHelper()->getRepository(Execution::class)->count([
+            'job' => $this->job,
+            'replicas' => 1,
+        ]);
+        if ((1 + $count) > ServerUtility::get('BLENDER_PARALLEL_REPLICA_COUNT', 5)) {
```

Comment:
```suggestion
        if ($count >= ServerUtility::get('BLENDER_PARALLEL_REPLICA_COUNT', 5)) {
```
Should have the same effect and is IMO more readable. But that's highly subjective to me. What do you think? :D